package.json

```diff
@@ -144,20 +145,22 @@
       "packages/fluentui/*"
     ],
     "nohoist": [
-      "@fluentui/babel-make-styles/tslib",
-      "@fluentui/babel-make-styles/typescript",
```

Comment:
This was introduced to have TS4 in `@fluentui/babel-make-styles`, can be deleted now.